hack/run-integration-tests.sh

```diff
@@ -33,9 +33,10 @@ fi
 
 KREW_BINARY=$(readlink -f "${1:-$KREW_BINARY_DEFAULT}")  # needed for `kubectl krew` in tests
 if [[ ! -x "${KREW_BINARY}" ]]; then
-  echo "Did not find $KREW_BINARY. You need to build krew before running the integration tests"
+  echo "Did not find $KREW_BINARY. You need to build krew FOR ${goos}/${goarch} before running the integration tests."
   exit 1
 fi
 export KREW_BINARY
 
-go test -v ./...
+"${SCRIPTDIR}/ensure-kubectl-installed.sh"
```

Comment:
That one puzzles me. You previously made a strong point against installing `kubectl` in this script. Quoting https://github.com/kubernetes-sigs/krew/issues/210#issue-454342289:

> (...)  since the name of run-integration-tests.sh doesn't make it clear that there's kubectl installation in there, it shouldn't install kubectl.

Calling that script here somehow defeats the purpose of the script, don't you think?

Reply:
You're right, this is me being an idiot and accepting both commits in merge-conflict.